jac-uk/apply
# Hotfix: #1798 Remove optional style in TextField component

## What's included?
Remove the optional style in the `TextField` component for now. The style will be brought back in another PR until we review which fields are required or optional and update the code accordingly. 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
1. Go to `Sign in` page and check if there is no optional text in the email and password label.
2. Go to `Reset your password` page and check if there is no optional text in the email label.

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
Include screen grabs, notes etc.

---
PREVIEW:DEVELOP
